src/workflows/runCRISPRCasFinderl_array.cwl

```diff
@@ -0,0 +1,60 @@
+cwlVersion: v1.2
+class: Workflow
+label: runCRISPRCasFinder
+doc: |
+      Implementation of CRISPRCasFinder processing for multiple input fasta files.
+
+requirements:
+  SubworkflowFeatureRequirement: {}
+  ResourceRequirement:
+    coresMin: 1
+    ramMin: 2000
+  ScatterFeatureRequirement: {}
+
+inputs:
+  sequences:
+    type: File[]
+    format: edam:format_1929
+    label: input Fasta file to analyze
+  soFile:
+    type: string?
+    label: path to the sel392v2.so file, required by vmatch
+    default: /opt/CRISPRCasFinder/sel392v2.so
+  casDefinition:
+    type: string?
```

Comment:
This one could be an Enum instead -> https://www.commonwl.org/user_guide/misc/#enum-inputs